### PR TITLE
KAFKA-7097 VerifiableProducer does not work properly with --message-create-time argument

### PR DESCRIPTION
**Pull request by tedyu**
Currently create time is interpreted as integer.

This PR makes the tool accept long values.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: tools/src/main/java/org/apache/kafka/tools/VerifiableProducer.java [hunk 1]
```diff
@@ -215,7 +215,7 @@ public static VerifiableProducer createFromArgs(ArgumentParser parser, String[]
         int throughput = res.getInt("throughput");
         String configFile = res.getString("producer.config");
         Integer valuePrefix = res.getInt("valuePrefix");
-        Long createTime = (long) res.getInt("createTime");
+        Long createTime = res.getLong("createTime");
```
Comment (omkreddy) [inline tools/src/main/java/org/apache/kafka/tools/VerifiableProducer.java:218]: we should also change the type to Long at Line no: 168

**Review by guozhangwang — APPROVED** [review]
Thanks @tedyu

**Comment by guozhangwang** [comment]
@tedyu Turns out this PR broke the system test, because `class VerifiableProducer(KafkaPathResolverMixin, VerifiableClientMixin, BackgroundThreadService)` is passing in `-1` as default `createTime`, and hence caused casting issue:

```
Exception in thread "main" java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.Long
at net.sourceforge.argparse4j.inf.Namespace.getLong(Namespace.java:136)
at org.apache.kafka.tools.VerifiableProducer.createFromArgs(VerifiableProducer.java:218)
at org.apache.kafka.tools.VerifiableProducer.main(VerifiableProducer.java:515)
```

this can be easily reproduced by run e.g. `ThrottlingTest.test_throttled_reassignment` test

**Comment by guozhangwang** [comment]
Could you fix it forward asap?

**Comment by tedyu** [comment]
Sorry for late response.
I am in China.

**Pull request merged**